### PR TITLE
Fixes for bonnie integration

### PR DESCRIPTION
**Pull request by daco101**
This PR adds the "is_top_level" field, and a convenience method for accessing all the stratifications.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1835
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `package.json` and `cqm-models.gemspec` has been updated N/A
- [x] Cqm-execution fixtures have been updated with the update_cqm_execution_fixtures.sh script inside server-scripts using this branch in the cqm-converter (will be tracked with https://jira.mitre.org/browse/BONNIE-1926)

**Bonnie Reviewer:**

Name: @losborne 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name: @mayerm94 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

### CODE REVIEW

File: app/models/cqm/cql_library.rb [hunk 1]
```diff
@@ -12,6 +12,11 @@ class CQLLibrary
     field :elm_annotations, type: Hash
     field :is_main_library, type: Boolean, default: false
 
+    # Currently this is only relevant for cql libraries that are included in composite measures,
+    # true indicates the library files are in the "top level" folder of the measure package, rather
+    # than coming from a component measure folder.
+    field :is_top_level, type: Boolean, default: true
```
Comment (losborne) [inline app/models/cqm/cql_library.rb]: Isn't the field above, `:is_main_library` for this purpose already?

File: app/models/cqm/measure.rb [hunk 2]
```diff
@@ -10,8 +10,6 @@ class Measure
     validates_inclusion_of :measure_scoring, in: %w[PROPORTION RATIO CONTINUOUS_VARIABLE COHORT]
     validates_inclusion_of :calculation_method, in: %w[PATIENT EPISODE_OF_CARE]
 
-    TYPES = %w[ep eh].freeze
```
Comment (losborne) [inline app/models/cqm/measure.rb]: Why was this removed?

File: app/models/cqm/measure.rb [hunk 3]
```diff
@@ -79,6 +77,10 @@ class Measure
 
     scope :by_measure_id, ->(id) { where(measure_id: id) }
 
+    def all_stratifications
```
Comment (losborne) [inline app/models/cqm/measure.rb]: this needs to be added to the mongoose schema as well

File: app/models/cqm/measure.rb (hunk 2 above)
Comment (zlister) [reply app/models/cqm/measure.rb]: This was leftover from the old model. It isn't used anymore.

File: app/models/cqm/cql_library.rb (hunk 1 above)
Comment (daco101) [reply app/models/cqm/cql_library.rb]: No, I believe only one library can ever be the main library.

File: app/models/cqm/measure.rb (hunk 3 above)
Comment (daco101) [reply app/models/cqm/measure.rb]: done.

File: app/models/cqm/cql_library.rb (hunk 1 above)
Comment (daco101) [reply app/models/cqm/cql_library.rb]: Resolved on slack.
